pom.xml

```diff
@@ -251,7 +251,7 @@
     <properties>
         <!--Carbon framework version-->
         <carbon.identity.framework.import.version.range>[5.0.0, 6.0.0)</carbon.identity.framework.import.version.range>
-        <carbon.identity.framework.version>5.5.0</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.18.91-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
Need to update the carbon identity framework version after merging: https://github.com/wso2/carbon-identity-framework/pull/3115